features/facebookImport/src/components/advertisingValueMiniStory/advertisingValueMiniStory.css

```diff
@@ -37,9 +37,9 @@
 }
 
 .detail-view .items {
+    font-family: "Jost Medium";
```

Comment:
Below, Jost Medium uses 500 weight, is that correct? So what this is doing is changing a weight for a font spec that has the same weight? Or a different one?

Reply:
I am not sure if I understand what you mean. But actually the weight should not be necessary here anymore, where we use the correct Jost Medium with the requested font-size

Reply:
The thing is that I see Jost Medium substituting for a few different font-weights, from 400 to 600. This is why I was asking if the correct one was 500. But never mind.